src/model.jl

```diff
@@ -38,31 +38,69 @@ end
 
 
 """
-    _make_bus_demand(case)
+    _make_bus_demand_weighting(case)
 
-Given a Case object, build a matrix of demand by (bus, hour) for this interval.
+Given a Case object, build a sparse matrix that indicates the weighting of each bus in 
+    each zone.
 """
-function _make_bus_demand(case::Case, start_index::Int, end_index::Int)::Matrix
+function _make_bus_demand_weighting(
+    case::Case, start_index::Int, end_index::Int
+)::SparseMatrixCSC
     bus_idx = 1:length(case.busid)
     bus_df = DataFrames.DataFrame(
-        name=case.busid, load=case.bus_demand, zone=case.bus_zone)
+        name=case.busid, load=case.bus_demand, zone=case.bus_zone
+    )
     zone_demand = DataFrames.combine(
-        DataFrames.groupby(bus_df, :zone), :load => sum)
+        DataFrames.groupby(bus_df, :zone), :load => sum
+    )
     zone_list = sort(collect(Set(case.bus_zone)))
-    num_zones = length(zone_list)
-    zone_idx = 1:num_zones
+    zone_idx = 1:length(zone_list)
     zone_id2idx = Dict(zone_list .=> zone_idx)
     bus_df_with_zone_load = DataFrames.innerjoin(bus_df, zone_demand, on=:zone)
     bus_share = bus_df[:, :load] ./ bus_df_with_zone_load[:, :load_sum]
     bus_zone_idx = Int64[zone_id2idx[z] for z in case.bus_zone]
     zone_to_bus_shares = sparse(
-        bus_zone_idx, bus_idx, bus_share)::SparseMatrixCSC
+        bus_zone_idx, bus_idx, bus_share
+    )::SparseMatrixCSC
+    return zone_to_bus_shares
```

Comment:
The `return` is not necessary here, Julia automatically returns the value of the last-evaluated expression.
See https://docs.julialang.org/en/v1/manual/functions/#The-return-Keyword

Reply:
Agree. But I will vote for keeping these `return` statements since they are more explicit and readable even for people not familiar with Julia.

Reply:
In that case, there are a few places where we should add them for consistency. See https://github.com/Breakthrough-Energy/REISE.jl/issues/127

Reply:
Okay, so we'll keep the `return` statements? I can add `return` statements to some of the functions that don't have them, like `_make_branch_map` and `_make_sets`.

Reply:
I think that would be a good idea, thanks for taking care of that.

Reply:
I added return statements to all functions where it made sense. I believe this should close #127